app/src/common/utils/index.js

```diff
@@ -40,3 +36,26 @@ export {
   daysBetween,
 } from './timeDateUtils';
 export { connectRouter } from './connectRouter';
+export {
+  createCheckPermission,
+  hasAccessToManagementSystem,
+  canCreateProject,
+  canDeleteProject,
+  canUpdateSettings,
+  canSeeSettings,
+  canCreateInternalUser,
+  canInviteInternalUser,
+  canAssignUnassignInternalUser,
+  canChangeUserRole,
+  canDeleteUser,
+  canSeeMembers,
+  canEditOwnAccount,
+  canDeleteLaunch,
+  canForceFinishLaunch,
+  canStartAnalysis,
+  canDeleteTestItem,
+  canMoveToDebug,
+  canMergeLaunches,
+  canWorkWithFilters,
+  canReadData,
+} from 'common/utils/permissions';
```

Comment:
As I see, this `index.js` grows bigger and bigger for no particlular reason. I don't think we should export everythink here, since `utils` is not a single module with interface, it's just a folder with different utils. I suggest removing this esport from here (other exports will be removed in future), and importing these functions directly from util file, for example:
`import { canMergeLaunches } from 'common/utils/permissions'`

This will make imports more specific.

@ArtsiomYeliseyenka what do you think?

Reply:
+1